tests/roles/run_tests/tasks/move_back.yml

```
@@ -9,6 +9,16 @@
       namespace: "{{ IRONIC_NAMESPACE }}"
       kubeconfig: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
 
+  # Install BMO in Source cluster
+  - name: Install Baremetal Operator in Source cluster
+    shell: "{{ BMOPATH }}/tools/deploy.sh true false {{ IRONIC_TLS_SETUP }} {{ IRONIC_BASIC_AUTH }} true"
```

The positional arguments [are deprecated](https://github.com/metal3-io/baremetal-operator/blob/ae34635a1e038ff42333bb3c952f5b9bfe301b83/tools/deploy.sh#L63). We should do it like for Ironic below.

```suggestion
    shell: "{{ BMOPATH }}/tools/deploy.sh -b {{ BMO_IRONIC_ARGS }}"
```

The variable that sets the arguments is already correctly set [here](https://github.com/metal3-io/metal3-dev-env/blob/c343d5648572da58cea1072d1e99b3a6b3632b1a/tests/roles/run_tests/vars/main.yml#L91) so we can use it directly. Just need to add `-b` for BMO.

Are you sure we should use `KUBECTL_ARGS` here btw? It is not used for Ironic below and I would assume it should be the same here.